packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,91 @@ path: /catalog/input-controls/form-fields/
+Use the `mdc-form-field` element to wrap any combination of adjacent _input_ and _label_ elements of MDC Web form controls, such as [MDC Checkbox](../mdc-checkbox) or [MDC Radio](../mdc-radio). Here's an example with MDC Checkbox:
 
 ```html
 <div class="mdc-form-field">
   <div class="mdc-checkbox">
-    <input type="checkbox"
-           id="my-checkbox"
-           class="mdc-checkbox__native-control"/>
+    <input type="checkbox" id="my-checkbox" class="mdc-checkbox__native-control"/>
     <div class="mdc-checkbox__background">
-      <svg class="mdc-checkbox__checkmark"
-           viewBox="0 0 24 24">
-        <path class="mdc-checkbox__checkmark-path"
-              fill="none"
-              stroke="white"
-              d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
-      </svg>
-      <div class="mdc-checkbox__mixedmark"></div>
+      ...
     </div>
   </div>
-  <label for="my-checkbox" id="my-checkbox-label">This is my checkbox</label>
+  <label for="my-checkbox">This is my checkbox</label>
 </div>
 ```
 
-> Note that MDC Form Field is **not** intended for cases where a label and input are already handled together by
-> another component's styles and logic. For example, JavaScript-enhanced instances of MDC Text Field already manage
-> a label and input together under their own root element.
-
-### RTL Support
-
-`mdc-form-field` is automatically RTL-aware, and will re-position elements within an RTL context.
-`mdc-form-field` will apply RTL styles whenever it, or its ancestors, has a `dir="rtl"` attribute.
-
-
-## JS Usage
-
-### Including in code
-
-#### ES2015
-
-```javascript
-import {MDCFormField, MDCFormFieldFoundation} from '@material/form-field';
-```
-
-#### CommonJS
-
-```javascript
-const mdcFormField = require('mdc-form-field');
-const MDCFormField = mdcFormField.MDCFormField;
-const MDCFormFieldFoundation = mdcFormField.MDCFormFieldFoundation;
-```
-
-#### AMD
-
-```javascript
-require(['path/to/mdc-form-field'], mdcFormField => {
-  const MDCFormField = mdcFormField.MDCFormField;
-  const MDCFormFieldFoundation = mdcFormField.MDCFormFieldFoundation;
-});
-```
-
-#### Global
+> _NOTE_: MDC Form Field is **not** intended for cases where a label and input are already handled together in a component's styles and logic. For example, [MDC Text Field](../mdc-textfield) already manages a label and input together under its own root element.
 
-```javascript
-const MDCFormField = mdc.formField.MDCFormField;
-const MDCFormFieldFoundation = mdc.formField.MDCFormFieldFoundation;
-```
+### JavaScript Instantiation
 
-### Instantiation
+If you are using MDC Form Field with an MDC Web component that has a [ripple](../mdc-ripple) effect, you can instantiate `MDCFormField` and set its [`input` property](#MDCFormField-properties-and-methods) to activate the ripple effect upon interacting with the label. Here is an example with [MDC Checkbox](../mdc-checkbox):
```

Comment:
Change the anchor on the input property link to all-lowercase (`#mdcformfield-properties-and-methods`) to make it work